src/qibolab/platform.py

```diff
@@ -483,7 +432,7 @@ def create_coupler_pulse(self, coupler, duration=None, amplitude=None):
             pulse = replace(pulse, duration=duration)
         if amplitude is not None:
             pulse = replace(pulse, amplitude=amplitude)
-        return replace(pulse, channel=coupler.flux.name)
+        return replace(pulse)
```

Comment:
Same as above

Reply:
create* methods of platform no longer exist